flipperdevices/flipperzero-firmware
# [FL-1920] Fix NTAG emulation

# What's new

- Always read signature if reading version was successful

# Verification 

- Verify signature is saved for NTAG cards
- Verify correct NXP public key in Nfc tools Pro while emulating mifare ultralight

# Checklist (do not modify)
